### ISSUE TITLE
re.match expecting a string but getting a list

### ISSUE THREAD

**Issue by IkBenOlie5** [parafoxia/len8#2]
in handle_when_file
```py
matches = re.match(
        r"^([~?\.\/\w].*)\/(\w+\.pyw?)$|^(\w+\.pyw?)$", args.path
)
```
args.path is an instance of a list not a string. This causes len8 to no longer work...
if i pass multiple files into args.path
```
$ len8 -f file1.py file2.py
```
the list would be `["file1.py",  "file2".py]`.
(this is because of #1)

**Comment by IkBenOlie5** [comment]
Maybe just remove the entire -f option all together. Just use the path argument for everything...

**Comment by Jonxslays** [comment]
This issue is now resolved in the [v0.2.0](https://github.com/parafoxia/len8/releases/tag/v0.2.1) release. You can update to the latest version with `pip install -U len8`.
